### DISCUSSION TITLE
OSS deployed on ECS is creating spikes in AWS Config

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17951] [Q&A]
I have Dagster deployed on *AWS ECS* by following the <https://docs.dagster.io/deployment/guides/aws|Deploying Dagster to Amazon Web Services> guide, but I have a *cost problem*.
I am running schedules that trigger around 800 runs, where each creates a separate ECS task, which leads to a spike in the AWS Config.
Is there a resolution on how to bring the amount of created configurations down?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699879593033089?thread_ts=1699879593.033089&cid=C01U954MEER).*

**Comment by ammerzon** [comment]
Here is the corresponding screenshot from the AWS Cost Center
![CleanShot 2023-11-13 at 13 44 36@2x](https://github.com/dagster-io/dagster/assets/7319168/de5fb1ca-d584-474b-9b17-7a6bd69b74b4)
